src/Components/App/App.js

```diff
@@ -1,31 +1,59 @@
 import './App.scss';
-import {Component} from 'react'
-import {BrowserRouter} from 'react-router-dom'
-import Lobby from '../Lobby/Lobby.js'
-import emptyLogo from '../../Assets/emptyLogo.svg'
+import {Component} from 'react';
+import {BrowserRouter, Switch, Route, NavLink} from 'react-router-dom';
+import WelcomePage from '../WelcomePage/WelcomePage.js';
+import Lobby from '../Lobby/Lobby.js';
+import ReflectionForm from '../ReflectionForm/ReflectionForm.js';
+import ReflectionList from '../ReflectionList/ReflectionList.js';
+import FeelingRoom from '../FeelingRoom/FeelingRoom.js';
+import BreathingRoom from '../BreathingRoom/BreathingRoom.js';
+import fullLogo from '../../Assets/emptyLogo.svg';
 
 class App extends Component {
   constructor(props){
```

Comment:
Do we need props to be passed in here? I don't think it has an impact on our code currently, but I'm just wondering if we have future plans for props being passed in.